react/Reviews.tsx

```diff
@@ -478,7 +478,15 @@ function Reviews() {
   }))
 
   const getTimeAgo = (time: string) => {
-    const before = new Date(time)
+    const before = Date.UTC(
+      new Date(time).getUTCFullYear(),
+      new Date(time).getUTCMonth(),
+      new Date(time).getUTCDate(),
+      new Date(time).getUTCHours(),
+      new Date(time).getUTCMinutes(),
+      new Date(time).getUTCSeconds()
+    )
+
```

Comment:
``` suggestion
    const date = new Date(time)
    const before = Date.UTC(
      date.getUTCFullYear(),
      date.getUTCMonth(),
      date.getUTCDate(),
      date.getUTCHours(),
      date.getUTCMinutes(),
      date.getUTCSeconds()
    )
```
Rewrite it to avoid creating the "new " instance multiple times.

Reply:
yeah, good point, thank you, changed!